index.html

```diff
@@ -19,6 +21,53 @@
       </div>
     </header>
 
+    <div class="about-popup">
+      <div class="close-about">
```

Comment:
How about `about-popup-close-icon`? I think it's useful when related classes have the same prefix, here `about-popup`.